### PR TITLE
Chore/DDW-205 Remove remote logging

### PR DESCRIPTION
**Pull request by darko-mijic**
This PR:
- removes logging to the remote Papertrail server
- replaces the content of the Settings/Support page with instructions how to access FAQ, use problem reporting feature and download logs
- fixes cancellation of download when downloading logs when manually reporting a problem

TODO:
- [x] Remove options for sending logs from the initial setup sequence
- [x] Remove settings for sending logs 
- [x] Fix acceptance tests
- [x] Proofread new copy
- [x] Japanese translations

screen shot 2018-03-29 at 12 53 23
screen shot 2018-03-30 at 08 24 43

### CODE REVIEW

File: features/transactions-grouping.feature [hunk 1]
```diff
@@ -8,8 +8,7 @@ Feature: Transactions Grouping
   Background:
     Given I have selected English language
     And I have accepted "Terms of use"
-    And I agree to send logs to remote server
-    And I have a wallet
+    And I have a wallet with funds
```
Comment (darko-mijic) [inline features/transactions-grouping.feature]: I have no idea how this step worked before ;)

**Comment by darko-mijic** [comment]
@a-rukin can you take a look at the layout. We haven't designed this upfront.

**Comment by a-rukin** [comment]
38030113-0159dfde-3298-11e8-8870-d7b560e3fee0 png 1 114x619 pixels 2018-03-29 13-05-46
You can use same padding as before "1 numbered title" here zpl.io/2ppw682

File: features/transactions-grouping.feature (hunk 1 above)
Comment (nikolaglumac) [reply features/transactions-grouping.feature]: @darko-mijic it did not work at all - it is skipped (notice the `@skip` line on the top of the file) 😱

**Pull request merged**